chsdi/lib/validation/__init__.py

```diff
@@ -5,6 +5,9 @@
 from chsdi.models.bod import Topics
 
 
+OUTPUT_SRS = ('21781', '2056', '3857', '4326')
```

Comment:
:+1:

Reply:
these are Marc's commits. I did not adapt chsdi any further for web mercator. This is why you see so many tests failing.